### PR TITLE
Moving EC2 (2) resources to v1beta1 version#27

### LINKED ISSUE #27: Moving EC2 (2) resources to v1beta1 version
- [ ] [`aws_ec2_transit_gateway_peering_attachment_accepter`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ec2_transit_gateway_peering_attachment_accepter)
- [ ] [`aws_snapshot_create_volume_permission`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/snapshot_create_volume_permission)

extracted from upbound/official-providers#467

### PR DESCRIPTION
**Pull request by dverveiko**
### Description of your changes

Fixes #27 

- [x] [`aws_ec2_transit_gateway_peering_attachment_accepter`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ec2_transit_gateway_peering_attachment_accepter)
- [x] [`aws_snapshot_create_volume_permission`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/snapshot_create_volume_permission)

I have:

- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

`aws_ec2_transit_gateway_peering_attachment_accepter` - Manually and [Uptest](https://github.com/upbound/provider-aws/actions/runs/3488191641)
`aws_snapshot_create_volume_permission` - Mannually and [Uptest](https://github.com/upbound/provider-aws/actions/runs/3548179104)

### CODE REVIEW

**Comment by dverveiko** [comment]
/test-examples="transitgatewaypeeringattachmentaccepter.yaml"

**Comment by dverveiko** [comment]
/test-examples="exaples/ec2/transitgatewaypeeringattachmentaccepter.yaml"

**Comment by dverveiko** [comment]
/test-examples="examples/ec2/transitgatewaypeeringattachmentaccepter.yaml"

**Comment by dverveiko** [comment]
/test-examples="examples/ec2/transitgatewaypeeringattachmentaccepter.yaml"

**Comment by dverveiko** [comment]
/test-examples="examples/ec2/transitgatewaypeeringattachmentaccepter.yaml"

**Comment by dverveiko** [comment]
/test-examples="examples/ec2/transitgatewaypeeringattachmentaccepter.yaml"

**Comment by dverveiko** [comment]
/test-examples="examples/ec2/transitgatewaypeeringattachmentaccepter.yaml"

**Comment by dverveiko** [comment]
/test-examples="examples/ec2/transitgatewaypeeringattachmentaccepter.yaml"

File: examples/ec2/snapshotcreatevolumepermission.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: SnapshotCreateVolumePermission
+metadata:
+  annotations:
+    meta.upbound.io/example-id: ec2/v1beta1/snapshotcreatevolumepermission
+    upjet.upbound.io/manual-intervention: "Problem on deletion. EBSSnapshot deleting beforeSnapshotCreateVolumePermission, what leads to stuck deletion proccess"
```
Comment (turkenh) [inline examples/ec2/snapshotcreatevolumepermission.yaml:6]: @dverveiko I am wondering if we could avoid this manual intervention using `post-assert` [hooks](https://github.com/upbound/uptest#hooks)? You can check [this](https://github.com/upbound/platform-ref-gcp/blob/6208fdfcd070b1ff01ba2ff2825a12f12852aa6d/examples/cluster-claim.yaml#L7) as an example.

**Comment by turkenh** [comment]
@dverveiko, you will need to sign the CLA.

**Comment by dverveiko** [comment]
/test-examples="examples/ec2/snapshotcreatevolumepermission.yaml"

**Comment by dverveiko** [comment]
/test-examples="examples/ec2/snapshotcreatevolumepermission.yaml"

**Comment by dverveiko** [comment]
/test-examples="examples/ec2/snapshotcreatevolumepermission.yaml"

**Comment by dverveiko** [comment]
/test-examples="examples/ec2/snapshotcreatevolumepermission.yaml"

**Comment by dverveiko** [comment]
/test-examples="examples/ec2/snapshotcreatevolumepermission.yaml"

**Review by turkenh — APPROVED** [review]
Looking great, thank you @dverveiko 🙌

**Pull request merged**